es/ae/cli.js

```diff
@@ -17,13 +17,13 @@
 
 import Ae from './'
 import Account from '../account/memory'
-import Chain from '../chain/epoch'
+import EpochChain from '../chain/epoch'
```

Comment:
This rename is unnecessary

Reply:
Not necessary, but it becomes consistent with the import of `EpochContract`. You have base Chain stamp and one connected to Epoch, may be difficult to understand which one is used currently without checking of import path.